### ISSUE TITLE
[Coherence][HIGH] Ch7 saturation diagnosis contradicted by H3 arms' own hyperparameters

### ISSUE THREAD

**Issue by kwojdalski** [kwojdalski/masters_thesis#715]
**Severity:** HIGH
**Source:** thesis-coherence-auditor run, 2026-09-02. Independently verified.

## What the thesis says

`thesis/qmd/src/07-02-limitations-and-future-research.qmd:148-151`:

> "Two properties of the setup make this diagnosable rather than mysterious. The arms that failed share the log-return objective or a small observation space, while the Differential Sharpe arms at the same budget trained normally, and alternative reward scalings tried during development did not recover the log-return arm."

## What the data says

All three Hypothesis 3 arms use the Differential Sharpe reward. Verified directly:

```
thesis/qmd/results/pooled_td3_h3_features_minimal/latest_finished/hyperparams.json
  reward_type = differential_sharpe, reward_eta = 0.01
thesis/qmd/results/pooled_td3_h3_features_selected/latest_finished/hyperparams.json
  reward_type = differential_sharpe, reward_eta = 0.01
thesis/qmd/results/pooled_td3_h3_features_full/latest_finished/hyperparams.json
  reward_type = differential_sharpe, reward_eta = 0.01
```

This is consistent with the single-factor design note the thesis itself states at `06-02-robustness-assessment.qmd:182` ("All three arms share the H1 train.yaml and differ only in observation.yaml").

Consequences:

1. The 33-feature ("full") arm satisfies **neither** disjunct of "the log-return objective or a small observation space". It uses DSR, and it has the **largest** observation space in the comparison (33 features vs. 10 for the arm that trained successfully).
2. "the Differential Sharpe arms at the same budget trained normally" is directly falsified: two of the three DSR arms in H3 (minimal and full) did not train. The thesis says so itself at `06-02-robustness-assessment.qmd:227-231` -- "the full 33-feature arm records turnover of \$4.7 \times 10^{-5}\$, also with zero position changes, and sits fully short."

## Why it matters

The incorrect diagnosis is used to justify a specific remediation ("A systematic sweep over reward scaling, actor learning rate, and exploration noise..."). Since reward is held constant across all three H3 arms, reward choice cannot explain the H3 failure pattern at all. Chapter 6's own hedge is the defensible statement and Chapter 7 currently overreaches past it.

An examiner who opens `hyperparams.json` -- or simply reads Section 6.3's own design note -- finds the thesis contradicting data it presented three sections earlier.

## Proposed fix

Either drop the "share the log-return objective or a small observation space" generalization entirely, or restrict it explicitly to the two arms it actually fits (Hypothesis 4's log-return arm and Hypothesis 3's minimal arm), and state plainly that the full-feature arm is an unexplained exception -- which is what Section 6.3 already says:

> "what unstable optimization landing arbitrarily looks like rather than a systematic response to feature count" (`06-02-robustness-assessment.qmd:236-240`)

**Cost:** a paragraph rewrite in `07-02-limitations-and-future-research.qmd`.
